recipes/mongo-c-driver/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.1)
+project(mongo-c-driver-tester C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(MONGO-C-DRIVER_SHARED)
+    message(STATUS "Testing shared mongoc")
+else()
+    message(STATUS "Testing static mongoc")
+    add_definitions (-DBSON_STATIC)
```

Comment:
Should ```conan``` provide ```BSON_STATIC```, ```MONGOC_STATIC```variables?

Reply:
yes, those definitions should be added in `package_info()`

Reply:
addressed in n-bes#1